Orange/widgets/visualize/owmosaic.py

```diff
@@ -294,7 +294,12 @@ def initCombos(self, data):
         self.attr3Combo.addItem("(None)")
         self.attr4Combo.addItem("(None)")
 
-        for attr in data.domain:
+        for attr in chain(data.domain.attributes, data.domain.metas):
+            if attr.is_discrete:
+                for combo in [self.attr1Combo, self.attr2Combo, self.attr3Combo, self.attr4Combo]:
+                    combo.addItem(self.icons[attr], attr.name)
+
+        for attr in data.domain.metas:
```

Comment:
Are you sure this is ok? Won't `domain.metas` be included twice now? Something must have changed since the rebase.